zuul.d/molecule.yaml

```diff
@@ -1,305 +1,354 @@
-    files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/local_env_vm/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
 - job:
+    files:
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/openshift_login/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
     name: cifmw-molecule-openshift_login
     nodeset: centos-9-crc-xl
     parent: cifmw-molecule-base
     vars:
       TEST_RUN: openshift_login
-    files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/openshift_login/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
 - job:
+    files:
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/openshift_provisioner_node/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
     name: cifmw-molecule-openshift_provisioner_node
+    nodeset: centos-9-crc-xl
     parent: cifmw-molecule-base
     vars:
       TEST_RUN: openshift_provisioner_node
-    files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/openshift_provisioner_node/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
 - job:
+    files:
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/openshift_setup/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
     name: cifmw-molecule-openshift_setup
     nodeset: centos-9-crc-xl
     parent: cifmw-molecule-base
     vars:
       TEST_RUN: openshift_setup
-    files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/openshift_setup/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
 - job:
+    files:
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/operator_build/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
     name: cifmw-molecule-operator_build
     parent: cifmw-molecule-base
     vars:
       TEST_RUN: operator_build
-    files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/operator_build/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
 - job:
+    files:
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/operator_deploy/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
     name: cifmw-molecule-operator_deploy
-    nodeset: centos-9-crc-xl
     parent: cifmw-molecule-base
     vars:
       TEST_RUN: operator_deploy
-    files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/operator_deploy/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
 - job:
+    files:
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/os_must_gather/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
     name: cifmw-molecule-os_must_gather
-    parent: cifmw-molecule-base-crc
```

Comment:
we can keep this parent `cifmw-molecule-base-crc` otherwise relevant molecule job will fail.

Reply:
keeping